builders/interact/substrate-api/wallets/subwallet.md

```diff
@@ -0,0 +1,111 @@
+
+1. Search for `Dancebox`
+2. Toggle the slider on to enable support for Dancebox
+
+![Toggle support for the Dancebox Testnet](/images/builders/interact/substrate-api/wallets/Subwallet/Subwallet-6.png)
+
+If you have a balance of Dancebox tokens, you'll see your account balance on the homepage of the Subwallet wallet. By default, all balances are hidden in Subwallet, but if you press the eye icon you can toggle balance visibility.
+
+![See your testnet account balances in Subwallet](/images/builders/interact/substrate-api/wallets/Subwallet/Subwallet-7.png)
+
+## Connecting Subwallet to Your Substrate ContainerChain {: #connecting-Subwallet-to-your-substrate-containerchain }
+
+To configure Subwallet for your Substrate ContainerChain, press the **More Options** icon in the upper left corner. Then click **Manage networks**. Press the **+** icon. On the following page, you'll then be prompted to enter the network details for your ContainerChain. For demonstration purposes, the Tanssi Dancebox Parachain is used here, but you can substitute these details for your own ContainerChain. To add your ContainerChain to Subwallet, take the following steps: 
+
+1. Paste in the WSS URL of your ContainerChain. Other parameters like the parachainId and token decimals may be auto populated
+2. Provide a name for your ContainerChain
+3. Press **Save**
+
+![Add Network in Subwallet](/images/builders/interact/substrate-api/wallets/Subwallet/Subwallet-8.png)
+
+## Connecting to Polkadot.js {: #connecting-to-polkadotjs}
+
+To connect your Substrate ContainerChain to Polkadot.js Apps, first head to [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffraa-dancebox-rpc.a.dancebox.tanssi.network#/accounts){target=_blank}. In this example, Polkadot.js Apps is connected to the Dancebox TestNet, but you can point Polkadot.js to your ContainerChain by clicking on the network dropdown and filling in the WSS endpoint of your ContainerChain in the **custom endpoint** field.
+
+![Connect to Polkadot.js Apps](/images/builders/interact/substrate-api/wallets/Subwallet/Subwallet-9.png)
+
+The Subwallet extension will prompt you to select the accounts you'd like to use with Polkadot.js Apps. If it doesn't automatically pop up, you can open the Subwallet extension and click on the **Connected / Disconnected** icon next to your account at the top. To configure Subwallet to correctly interface with your ContainerChain on Polkadot.js Apps, you should take the following steps:
```

Comment:
```suggestion
The Subwallet extension will prompt you to select the accounts you'd like to use with Polkadot.js Apps. If it doesn't automatically pop up, you can open the Subwallet extension and click on the **Connected** icon next to your account at the top. To configure Subwallet to correctly interface with your ContainerChain on Polkadot.js Apps, you should take the following steps:
```

Reply:
When you hover over the connected icon it says "Connected" but it doesn't ever say "Disconnected" and when items are in bold it indicates that its a UI element, so I think just using "Connected" here is fine